src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -708,6 +711,19 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
 
             var grpcEndpointUrl = await _dashboardEndpointProvider.GetResourceServiceUriAsync(context.CancellationToken).ConfigureAwait(false);
 
+            if (dashboardAuthenticationOptions.Value.AllowUnsecureTransport.GetValueOrDefault(false))
+            {
+                // Configure for insecure running.
+                context.EnvironmentVariables["DISABLEAUTHPLACEHOLDER"] = true;
```

Comment:
Who respects these env vars?

Do they also need to go in the `StartDashboardAsDcpExecutableAsync` method below?

Reply:
This code has been removed for now.